pkg/middleend/middleend.go

```diff
@@ -51,12 +51,7 @@ func (s *Server) CreateEncryptedVolume(_ context.Context, in *pb.CreateEncrypted
 		log.Print(msg)
 		return nil, status.Errorf(codes.InvalidArgument, msg)
 	}
-	params1 := models.AccelCryptoKeyCreateParams{
```

Comment:
I think since we have a switch case, we should remove the regex above

Reply:
It turned out that sw accel framework which is used by default supports only [AES_XTS_128 and AES_XTS_256](https://github.com/spdk/spdk/blob/535543d4f165051770fd011cb90973519ac167e2/lib/accel/accel_sw.c#L667) I provided a validation and tests to check for those ciphers.